app/main.py

```diff
@@ -1,3 +1,21 @@
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self, name: str,
+                 description: str,
+                 weeks: int | float) -> None:
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    def __str__(self) -> str:
+        return self.description
+
+    @staticmethod
+    def days_to_weeks(days: int) -> int:
+        result = days / 7
+        return result if result == int(result) else int(result) + 1
+
+    @classmethod
+    def from_dict(cls, course_dict: dict) -> "OnlineCourse":
```

Comment:
At start of file you can import `from __future__ import annotations` and simply use a class as returning type 
`-> OnlineCourse `